bin/update_tarball-install.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+DEST=/usr/local/repo/tarball-install
+
+#############################
+### Mirror tarball-client ###
+#############################
+
+/usr/bin/lftp -c mirror                                              \
+  --delete                                                           \
+  --exclude-glob=osg-afs-client-*         `# Skip osg-afs-client`    \
+  --exclude-glob=*tarballs.rescue         `# Ignore rescue tarballs` \
+  --exclude-glob=osg-wn-client-latest.*   `# Ignore latest symlinks` \
+  --exclude=3.2                           `# Exclude old releases`   \
+  https://vdt.cs.wisc.edu/tarball-client/                            \
+  $DEST
+
+##############################
+### Create latest symlinks ###
+##############################
+
+# For each OSG release series directory
+for REL_DIR in "$DEST"/* ; do
```

Comment:
```suggestion
for REL_DIR in $DEST/* ; do
```

I mean, it's fine if you want to quote `$DEST` ...  but it's hard coded in the script and not based on external input, so there's really no need.